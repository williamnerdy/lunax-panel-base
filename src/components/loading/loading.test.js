import React from 'react';
import { shallow } from 'enzyme';
import Loading from './loading';

describe('<Loading />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
