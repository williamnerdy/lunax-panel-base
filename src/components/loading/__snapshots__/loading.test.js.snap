// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <svg
        class="spinner"
        height="65px"
        viewBox="0 0 66 66"
        width="65px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          class="path"
          cx="33"
          cy="33"
          fill="none"
          r="30"
          stroke-linecap="round"
          stroke-width="6"
        />
      </svg>,
      "className": "spinner-wrapper",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <circle
          class="path"
          cx="33"
          cy="33"
          fill="none"
          r="30"
          stroke-linecap="round"
          stroke-width="6"
        />,
        "class": "spinner",
        "height": "65px",
        "viewBox": "0 0 66 66",
        "width": "65px",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "class": "path",
          "cx": "33",
          "cy": "33",
          "fill": "none",
          "r": "30",
          "stroke-linecap": "round",
          "stroke-width": "6",
        },
        "ref": null,
        "rendered": null,
        "type": "circle",
      },
      "type": "svg",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <svg
          class="spinner"
          height="65px"
          viewBox="0 0 66 66"
          width="65px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            class="path"
            cx="33"
            cy="33"
            fill="none"
            r="30"
            stroke-linecap="round"
            stroke-width="6"
          />
        </svg>,
        "className": "spinner-wrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <circle
            class="path"
            cx="33"
            cy="33"
            fill="none"
            r="30"
            stroke-linecap="round"
            stroke-width="6"
          />,
          "class": "spinner",
          "height": "65px",
          "viewBox": "0 0 66 66",
          "width": "65px",
          "xmlns": "http://www.w3.org/2000/svg",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "class": "path",
            "cx": "33",
            "cy": "33",
            "fill": "none",
            "r": "30",
            "stroke-linecap": "round",
            "stroke-width": "6",
          },
          "ref": null,
          "rendered": null,
          "type": "circle",
        },
        "type": "svg",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
